[2025-04-18T12:54:15.344+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-04-18T12:54:15.389+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: weather_api_to_snowflake.load_weather_raw_to_snowflake manual__2025-04-18T12:53:59.496922+00:00 [queued]>
[2025-04-18T12:54:15.424+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: weather_api_to_snowflake.load_weather_raw_to_snowflake manual__2025-04-18T12:53:59.496922+00:00 [queued]>
[2025-04-18T12:54:15.430+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 2
[2025-04-18T12:54:15.467+0000] {taskinstance.py:2889} INFO - Executing <Task(SnowflakeOperator): load_weather_raw_to_snowflake> on 2025-04-18 12:53:59.496922+00:00
[2025-04-18T12:54:15.481+0000] {standard_task_runner.py:72} INFO - Started process 146 to run task
[2025-04-18T12:54:15.490+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'weather_api_to_snowflake', 'load_weather_raw_to_snowflake', 'manual__2025-04-18T12:53:59.496922+00:00', '--job-id', '188', '--raw', '--subdir', 'DAGS_FOLDER/dbt_snowflake_pipeline.py', '--cfg-path', '/tmp/tmpg1xd3gvu']
[2025-04-18T12:54:15.503+0000] {standard_task_runner.py:105} INFO - Job 188: Subtask load_weather_raw_to_snowflake
[2025-04-18T12:54:15.615+0000] {task_command.py:467} INFO - Running <TaskInstance: weather_api_to_snowflake.load_weather_raw_to_snowflake manual__2025-04-18T12:53:59.496922+00:00 [running]> on host b322df8cb9fb
[2025-04-18T12:54:15.936+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='weather_api_to_snowflake' AIRFLOW_CTX_TASK_ID='load_weather_raw_to_snowflake' AIRFLOW_CTX_EXECUTION_DATE='2025-04-18T12:53:59.496922+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-04-18T12:53:59.496922+00:00'
[2025-04-18T12:54:15.956+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-04-18T12:54:16.015+0000] {sql.py:278} INFO - Executing: 
            CREATE TABLE IF NOT EXISTS STAGING.WEATHER_RAW (
                    city STRING,
                    timestamp TIMESTAMP,
                    temperature FLOAT,
                    humidity FLOAT,
                    pressure FLOAT,
                    description STRING
            );
        

                INSERT INTO STAGING.WEATHER_RAW (
                    city, timestamp, temperature, humidity, pressure, description
                )
                VALUES (
                    'Paris',
                    CURRENT_TIMESTAMP,
                    18.2,
                    42,
                    1012,
                    'clear sky'
                );
            

                INSERT INTO STAGING.WEATHER_RAW (
                    city, timestamp, temperature, humidity, pressure, description
                )
                VALUES (
                    'London',
                    CURRENT_TIMESTAMP,
                    16.25,
                    51,
                    1010,
                    'overcast clouds'
                );
            

                INSERT INTO STAGING.WEATHER_RAW (
                    city, timestamp, temperature, humidity, pressure, description
                )
                VALUES (
                    'New York',
                    CURRENT_TIMESTAMP,
                    11.17,
                    48,
                    1026,
                    'clear sky'
                );
            
[2025-04-18T12:54:16.070+0000] {base.py:84} INFO - Retrieving connection 'snowflake_conn'
[2025-04-18T12:54:16.101+0000] {base.py:84} INFO - Retrieving connection 'snowflake_conn'
[2025-04-18T12:54:16.106+0000] {connection.py:423} INFO - Snowflake Connector for Python Version: 3.14.0, Python Version: 3.11.11, Platform: Linux-5.10.76-linuxkit-x86_64-with-glibc2.36
[2025-04-18T12:54:16.111+0000] {connection.py:1226} INFO - Connecting to GLOBAL Snowflake domain
[2025-04-18T12:54:16.114+0000] {connection.py:1315} INFO - This connection is in OCSP Fail Open Mode. TLS Certificates would be checked for validity and revocation status. Any other Certificate Revocation related exceptions or OCSP Responder failures would be disregarded in favor of connectivity.
[2025-04-18T12:54:16.777+0000] {sql.py:553} INFO - Running statement: CREATE TABLE IF NOT EXISTS STAGING.WEATHER_RAW (
                    city STRING,
                    timestamp TIMESTAMP,
                    temperature FLOAT,
                    humidity FLOAT,
                    pressure FLOAT,
                    description STRING
            );, parameters: None
[2025-04-18T12:54:16.846+0000] {sql.py:562} INFO - Rows affected: 1
[2025-04-18T12:54:16.856+0000] {snowflake.py:447} INFO - Rows affected: 1
[2025-04-18T12:54:16.862+0000] {snowflake.py:448} INFO - Snowflake query id: 01bbc586-0001-5351-0001-2aee000160aa
[2025-04-18T12:54:16.867+0000] {sql.py:553} INFO - Running statement: INSERT INTO STAGING.WEATHER_RAW (
                    city, timestamp, temperature, humidity, pressure, description
                )
                VALUES (
                    'Paris',
                    CURRENT_TIMESTAMP,
                    18.2,
                    42,
                    1012,
                    'clear sky'
                );, parameters: None
[2025-04-18T12:54:17.180+0000] {sql.py:562} INFO - Rows affected: 1
[2025-04-18T12:54:17.193+0000] {snowflake.py:447} INFO - Rows affected: 1
[2025-04-18T12:54:17.200+0000] {snowflake.py:448} INFO - Snowflake query id: 01bbc586-0001-532c-0001-2aee0001415a
[2025-04-18T12:54:17.209+0000] {sql.py:553} INFO - Running statement: INSERT INTO STAGING.WEATHER_RAW (
                    city, timestamp, temperature, humidity, pressure, description
                )
                VALUES (
                    'London',
                    CURRENT_TIMESTAMP,
                    16.25,
                    51,
                    1010,
                    'overcast clouds'
                );, parameters: None
[2025-04-18T12:54:17.584+0000] {sql.py:562} INFO - Rows affected: 1
[2025-04-18T12:54:17.591+0000] {snowflake.py:447} INFO - Rows affected: 1
[2025-04-18T12:54:17.595+0000] {snowflake.py:448} INFO - Snowflake query id: 01bbc586-0001-5351-0001-2aee000160ca
[2025-04-18T12:54:17.598+0000] {sql.py:553} INFO - Running statement: INSERT INTO STAGING.WEATHER_RAW (
                    city, timestamp, temperature, humidity, pressure, description
                )
                VALUES (
                    'New York',
                    CURRENT_TIMESTAMP,
                    11.17,
                    48,
                    1026,
                    'clear sky'
                );, parameters: None
[2025-04-18T12:54:17.992+0000] {sql.py:562} INFO - Rows affected: 1
[2025-04-18T12:54:18.000+0000] {snowflake.py:447} INFO - Rows affected: 1
[2025-04-18T12:54:18.003+0000] {snowflake.py:448} INFO - Snowflake query id: 01bbc586-0001-532c-0001-2aee0001415e
[2025-04-18T12:54:18.298+0000] {connection.py:810} INFO - closed
[2025-04-18T12:54:18.403+0000] {connection.py:816} INFO - No async queries seem to be running, deleting session
[2025-04-18T12:54:18.513+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-04-18T12:54:18.518+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=weather_api_to_snowflake, task_id=load_weather_raw_to_snowflake, run_id=manual__2025-04-18T12:53:59.496922+00:00, execution_date=20250418T125359, start_date=20250418T125415, end_date=20250418T125418
[2025-04-18T12:54:18.643+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-04-18T12:54:18.722+0000] {taskinstance.py:3895} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-04-18T12:54:18.734+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
